appinventor/components/src/com/google/appinventor/components/runtime/ComponentContainer.java

```diff
@@ -52,4 +52,5 @@ public interface ComponentContainer {
 
   int Height();
 
+
```

Comment:
Since this whitespace-only change is the sole modification in this file, please revert the file to match ucr.